morpheushelper/cogs/mod.py

```diff
@@ -242,11 +242,11 @@ async def roles_mute(self, ctx: Context, role: Role):
 
     @commands.command()
     @guild_only()
-    async def report(self, ctx: Context, member: Member, *, reason: str):
+    async def report(self, ctx: Context, member: Union[Member, int], *, reason: str):
         """
         report a member
         """
-
+        member: Member = await self.get_user(ctx.guild, member)
```

Comment:
Maybe another var name like user makes more sense because it doesn't overload the method param.
Also the type is a `Member` or a `User` so the typing should be like this:
```python
user: Union[Member, User] = await self.get_user(ctx.guild, member)
```

Reply:
Is there anything wrong with overloading the methods parameters?I had actually done this on purpose

Reply:
It has also been done in the ban method as follows. Line 479

Reply:
I would argue that sometimes it's not the most intuitive way... But if it was on pupose feel free to ignore my comment or just think about it a bit

Reply:
Same with all the other functions. As `member` is not necessarily a Member I would change it to `user` and - of course - change the type hint